debug/physmem.py

```diff
@@ -121,3 +124,61 @@ def dump_tlb(self):
                 continue
             rows.append(row)
         ptable(rows, fmt="rrll", header=True)
+
+
+class KernelLog():
+    def invoke(self):
+        messages, klog = self.load_klog()
+        self.dump_general_info(len(messages), klog)
+        self.dump_kernel_logs(messages)
+
+    def format_message(self, data):
+        message = data['kl_format'].string()
+        arguments = data['kl_params']
+        # If there is % escaped it is not parameter.
+        number_of_parameters = message.count('%') - 2 * message.count('%%')
```

Comment:
For using function gdb.execute print we need to give it correct number of parameters (line 147)
and I use it for so I don't need to check if parameter is address or value